### ISSUE TITLE
Invalid config after upgrade

### ISSUE THREAD

**Issue by xconverge** [Tautulli/Tautulli#1395]
Just upgraded from 2.6.8 to 2.6.9 with a docker image and my config doesnt seem to be parsing

```
Unable to initialize Tautulli due to a corrupted config file. Exiting...
Unable to initialize Tautulli due to a corrupted config file. Exiting...
Unable to initialize Tautulli due to a corrupted config file. Exiting...
Unable to initialize Tautulli due to a corrupted config file. Exiting...
Unable to initialize Tautulli due to a corrupted config file. Exiting...
Unable to initialize Tautulli due to a corrupted config file. Exiting...
Unable to initialize Tautulli due to a corrupted config file. Exiting...
```

My config file is still version 17 and it looks like the new version is 18

If I delete my config and restart, it starts up fine with the default config. 

I cannot find any obvious differences in the config.py changelog

Pretty sure this has nothing to do with my docker setup/image since it loads fine when I change out config.ini

Is there anything else I can provide to be of help? I dont really want to strip my config.ini of sensitive data...

I am not worried about getting up and running, I could easily recreate my config off of the new baseline, but wanted to at least give a heads up incase others run into this

**Comment by JonnyWong16** [comment]
Just import the config file in the settings.

**Comment by xconverge** [comment]
Going through the new setup and then importing my version 17 config causes the corruption message again

**Comment by JonnyWong16** [comment]
Then your config file is actually corrupted.

I can't tell you anything more than that because you have not filled in the issue template will all the required info.

**Comment by xconverge** [comment]
I will close the issue then. I am going to make the change I need and move on with life. I wanted to give you a heads up and that is done now. 

I figured you would like to know in a timely manner at the time of a release, in the case of it being an inherent bug and not a one-off user (which you would have a better idea of based on the changed between releases vs me)

I understand the need for the template and I thought I would sidestep it in this case, but understand how annoying that can be for the maintainer, sorry. Without the application starting, no logs are being generated in tautulli.log so I did not provide it.

**Comment by JonnyWong16** [comment]
I am looking in to it but I would at least like to know things like operating system, python version, etc. That type of info would help me narrow it down.

Logs can be access on the file system directly without the web interface. That would tell me if there are issues in the update process.

**Comment by xconverge** [comment]
I looked at the logs in the filesystem and they didnt even get touched/updated since the last successful run with a known good config. So 0 relevant logs to attach unfortunately

```
Git Branch: | master
-- | --
Git Commit Hash: | v2.6.9
Configuration File: | /config/config.ini
Database File: | /config/tautulli.db \| Status
Log File: | /config/logs/tautulli.log
Backup Directory: | /config/backups
Cache Directory: | /config/cache
Export Directory: | /config/exports
Newsletter Directory: | /config/newsletters
Arguments: | ['--datadir', '/config']
Platform: | [Docker] Linux 5.11.4-051104-generic (#202103071231 SMP Sun Mar 7 12:53:00 UTC 2021)
System Timezone: | America/Los_Angeles (UTC-0700)
Python Version: | 3.8.8 (default, Mar 15 2021, 13:10:14)  [GCC 10.2.1 20201203]
```

(linuxserver image)

**Comment by JonnyWong16** [comment]
I managed to figure it out. Fixed in v2.6.10.

**Comment by xconverge** [comment]
Yep I was just about to propose changing version in config.ini to 17 and set my home_cards value to what I had and seeing if it upgrades successfully.

Thanks for looking into it, this sort of timely fix before more people upgrade is better for everyone involved which is why I rushed to post without the full template.
